language/stdlib/modules/Testnet.move

```diff
@@ -21,5 +21,97 @@ module Testnet {
         Transaction::assert(Signer::address_of(account) == 0xA550C18, 0);
         IsTestnet{} = move_from<IsTestnet>(0xA550C18);
     }
+
+    // **************** SPECIFICATIONS ****************
+
+    /*
+       This modules is used by Genesis, LibraAccount and Unhosted as part of
+       the initialization and to check whether the transaction is on the
+       testnet.
+    */
+
+    /// # Module Specification
+
+    spec module {
+        /// Verify all functions in this module, including private ones.
+        pragma verify = true;
+
+        /// Returns the association root address.
+        define spec_root_address(): address { 0xA550C18 }
+
+        /// Returns true if the testnet has been initialized.
+        define spec_testnet_initialized(): bool {
```

Comment:
I think we should call similar functions "spec_is_initialized" in all modules, consistently.

Please add a comment that this doesn't follow our conventions because it can be reversed by remove_testnet.